### PR TITLE
Fix ESLinting

### PR DESCRIPTION
**Pull request by matijs**
Change the "lint" and "lint:fix" scripts in package.json so ESLint actually lints all it needs to lint.

-  Auto fix linting violations
-  Manually fix linting violations
-  Override linting in search_field.test.tsx to keep test passing, but add a comment that the test needs to be rewritten.

### CODE REVIEW

File: src/components/input_fields/__tests__/search_field.test.tsx [hunk 1]
```diff
@@ -50,9 +50,12 @@ describe('The search field component', () => {
         props.staticField = true;
         props.children = 'a child';
         wrapper.setProps(props);
-        props['inputChildren'] = props['children'];
+        // This test needs to be rewritten
```
Comment (xli12) [inline src/components/input_fields/__tests__/search_field.test.tsx]: Is that possible to elaborate the reason why this test needs to be rewritten?

File: src/jestsetup.tsx [hunk 2]
```diff
@@ -1,5 +1,7 @@
 // Copyright (c) 2018 Ultimaker B.V.
+/* eslint-disable */
```
Comment (xli12) [inline src/jestsetup.tsx]: `jestsetup.tsx` exists in multiple applications. Just noticed that for some applications `/* eslint-disable */` is not needed, as the dependencies are installed not only for dev, but also for production. Just want to check if there's any (in)consistency in these projects...

Comment (matijs) [reply src/jestsetup.tsx]: I've looked into this… https://github.com/Ultimaker/stardust-web-components/pull/31/files#diff-e4403a877d80de653400d88d85e4801aR5 seems to be a nicer way of solving this without having to use inline overrides.

File: src/components/input_fields/__tests__/search_field.test.tsx (hunk 1 above)
Comment (matijs) [reply src/components/input_fields/__tests__/search_field.test.tsx]: I think it's not a very well written test. The whole file is a bit meh. In the `beforeEach` there's even an assertion…

**Pull request merged**